src/theme.ts

```diff
@@ -2,16 +2,20 @@ import { css } from 'styled-components';
 
 const palette = {
   red: "#ca2026",
-  danger: "#c2002f", // dark red
+  danger: "#c2002f",
+  darkRed: "#c22032",
```

Comment:
I've asked Devika if we can reuse existing colors rather than add the new green and red.